Clinical-Genomics/scout
# Add rerunner integration

This PR integrates a service for toggling a reanalysis of a sample using updated pedigree metadata

**How to test**:
1. goto cust000/643594, verify that "correct metadata" button is absent
2. set RERUNNER_HOST and/or RERUNER_PORT and RERUNNER_API_KEY  variables in configuration
3. goto previous pages and verify that button is present

**Expected outcome**:
The correct metadata button should only be displayed when the `RERUNNER_HOST` is set in the config

**Review:**
- [x] tests executed by MJ, CR
